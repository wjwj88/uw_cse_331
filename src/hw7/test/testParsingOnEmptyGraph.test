# Tests building an empty graph hw7

# Load emptyGraph.tsv
LoadGraph empty emptyGraph.tsv

# Print nodes in the graph 
ListNodes empty