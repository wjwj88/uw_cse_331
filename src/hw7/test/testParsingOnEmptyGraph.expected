# Tests building an empty graph hw7

# Load emptyGraph.tsv
loaded graph empty 

# Print nodes in the graph 
empty contains: